public/locales/en/translation.json

```diff
@@ -327,6 +327,7 @@
     "ack-list-item-3": "I declare under penalty of perjury that I am a US Citizen or National; or an Alien in satisfactory immigration status and permitted to work by the United States Citizenship and Immigration Service.",
     "ack-list-item-4": "I understand when submitting my request for benefits my submission is considered the same as my written signature.",
     "ack-label": "I agree and accept the statement above.",
+    "ack-feedback": "You must indicate your acceptance of the statement by checking the box before your certification can be submitted.",
```

Comment:
Can you add this to the Spanish file as well? It will make the Spanish easier to implement if the keys match.

Reply:
Oops, yes. Done!